cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -116,10 +116,24 @@ def _request_sub_recipe(self, inode):
             return None
         return data
 
-    def _migrate_version(self, recipe, format_version="1.0"):
+    @staticmethod
+    def _sanitize_format_version(recipe_data):
+        if "format_version" not in recipe_data:
+            format_version = "1.0"  # all recipes before we introduced versioning
+        elif len(recipe_data["format_version"].split(".")) > 2:
```

Comment:
maybe could add another check to account for something like `recipe_data["format_version"] = 2`? Not really necessary but covers more cases.